enrolments/tests/test_validators.py

```diff
@@ -12,7 +13,39 @@ def setUp(self):
                 Student(first_name="Buckets", last_name="Jr", role="Child"),
             ]
         )
+        Field.objects.bulk_create(
```

Comment:
insightful questions 🔬

Reply:
I agree, very important questions everyone should be asking themselves